queries.yaml

```diff
@@ -903,7 +905,7 @@ post_process:
   - fn: vectordatasource.transform.update_min_zoom
     params:
       source_layer: water
-      start_zoom: 4
+      start_zoom: 2
```

Comment:
This could be zoom 4 like before, but future proofing as it's low cost and if the other config is changed to 2 from 4, this would already allow it to work